cmake/modules/FindStdFilesystem.cmake

```diff
@@ -28,7 +28,6 @@ set(_std_filesystem_already_included FALSE)
 foreach(library
     ""
     "stdc++fs"
-    "c++experimental"
```

Comment:
I was going to ask if `FindStdFilesystem.cmake` is still needed -- it is.  gcc 8 still requires specifying `-lstdc++fs` and clang is probably similar.

Reply:
indeed. if we can/want to change the required version of clang, i will update this part accordingly.